style.css

```diff
@@ -0,0 +1,168 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap");
+
+:root {
+  --black: #1c1a19;
+  --dark-gray: #3c3a39;
+  --orange: #ff6b00;
+  --light: #fff5e1;
+}
+
+body {
+  background-color: var(--black);
+  margin: 0;
+  width: 100%;
+  min-width: 395px;
+}
+
+.appBar {
+  height: 17vh;
+  background-color: var(--dark-gray);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.appBar a {
+  text-decoration: none;
+  margin: 0;
+  padding: 0;
+}
+
+#menu-icon {
+  width: 24px;
+  right: 25px;
+  margin-right: 25px;
+  margin-bottom: 25px;
+}
+
+.logo-text {
+  width: 250px;
+  font-size: 3.5vh;
+  color: white;
+  margin-left: 8%;
+  margin-bottom: 20px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+}
+
+.section-headline {
+  height: 100vh;
+  margin-left: 24px;
+  background-image: url("images/Group.png");
+  background-repeat: no-repeat;
+  background-position: right bottom;
+}
+
+.heading {
+  /* Primary Text */
+  height: 124px;
+  margin-top: 120px;
+  color: var(--orange);
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10vh;
+  line-height: 64px;
+
+  /* or 114% */
+
+  letter-spacing: 0.37px;
+  margin-bottom: 0;
+}
+
+.sub-heading {
+  color: var(--light);
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 5vh;
+  line-height: 32px;
+
+  /* or 133% */
+}
+
+.description {
+  height: 123px;
+  margin-top: 12px;
+  color: white;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.section-social-media {
+  margin-left: 5px;
+  padding: 0;
+}
+
+.links {
+  margin: 0;
+  padding-left: 0;
+  display: flex;
+  list-style: none;
+}
```

Comment:
- [ ] Some icons are not horisontaly centred,  please add `align-items: center` to properly center them.

Reply:
Resolved. Aligned icons horizontally at line 113 in style.css